indico/util/placeholders.py

```diff
@@ -191,6 +191,7 @@ def serialize(cls, **kwargs):
         return dict(super().serialize(), parametrized=True, params=params)
 
 
+@make_interceptable
```

Comment:
It's already getting stuff from a signal, what is the need to intercept the function itself? Removing some default placeholders?

Reply:
This change is because we are using the same class RegistrantsListToBadgesPDF for printing with the accreditation module, which is a module similar to the registration module. This class calls `placeholders = get_placeholders('designer-fields')` but we want it to call get_placeholders(''accreditation-designer-fields'). (It will probable be changed later again when we refactor this module)

Reply:
Yes, this will work, thanks. But since we were not subclassing the class I'll set it on the object 
pdf = pdf_class(self.template, config_params, None, accreditations, include_accompanying_persons=False)
pdf.placeholders_context = 'accreditation-designer-fields'

Reply:
Yes, that should be fine as well. Can you update this PR to add my changes instead of the `make_interceptable` on this function?

Reply:
Okay

Reply:
Done

Reply:
so the `make_interceptable` here should not be needed anymore..

Reply:
Ah sorry I forgot to remove it. Removed now